pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "todo-app"
+version = "0.1.0"
+description = "A simple ToDo app for your DevOps apprenticeship"
+authors = []
+
+[tool.poetry.dependencies]
+python = "^3.7"
+flask = "^1.1.2"
+python-dotenv = "^0.14.0"
+requests = "^2.24.0"
+
+[tool.poetry.dev-dependencies]
```

Comment:
You should run `poetry add --dev pytest selenium`
And update your README to say to run tests via `poetry run pytest` instead of just `pytest`